relay-spans/src/span.rs

```diff
@@ -144,12 +41,18 @@ fn convert_from_otel_to_sentry_status(
     SpanStatus::Unknown
 }
 
-impl From<OtelSpan> for EventSpan {
-    fn from(from: OtelSpan) -> Self {
+/// Extension trait for OtelSpan.
+pub trait OtelSpanExt {
+    /// Transform an OtelSpan to an Sentry span.
+    fn to_sentry_span(self) -> EventSpan;
+}
+
+impl OtelSpanExt for OtelSpan {
```

Comment:
Any reason not to keep the `From` trait?

Reply:
i can't implement a foreign trait on a foreign trait, see
https://doc.rust-lang.org/error_codes/E0117.html

Reply:
Could just be a free standing function?

Reply:
if y'all prefer that, sure, idk much rust convention